docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,51 @@
+# Legacy -> UEFI Update
+
+This document describes the process of updating the platform's legacy SeaBIOS
```

Comment:
I see it more as `initial deployment` step, where we are installing Dasharo for the first time, from the legacy PC Engines firmware based on coreboot + SeaBIOS,

Reply:
How about this:
`This document describes the process of initial deployment of Dasharo from legacy firmware based on coreboot + SeaBIOS`
Should I also change the title to `Initial Dasharo deployment`?

Reply:
Maybe let's mention old firmware or PC Engines firmware and link https://pcengines.github.io/
Saying: `coreboot+SeaBIOS` or legacy firmware could be incorrectly understood. Please note we maintained the mainline and legacy lines on the PC Engines website. So we should refer to old firmware saying `PC Engines firmware` or `PC Engines coreboot firmware` with a link. @macpijan WDYT?

Reply:
`PC Engines coreboot firmware` with a link underneath sounds good to me, let's use this @PLangowski